tests/misc.sh

```diff
@@ -87,7 +87,7 @@ jexpect()
 
 test_check()
 {
-	if [ $* ]; then
+	if [ $* ] 2>/dev/null ; then
```

Comment:
I agree that this is not ideal. It might be a good idea to make this more sensible in a different way in a future commit.